src/Symfony/Component/Config/ConfigCacheInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config;
+
+/**
+ * Interface for ConfigCache
+ *
+ * @author Matthias Pigulla <mp@webfactory.de>
+ */
+interface ConfigCacheInterface
+{
+    /**
+     * Gets the cache file path.
+     *
+     * @return string The cache file path
+     */
+    public function __toString();
```

Comment:
Just to make sure I don't get you wrong: For example [here](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Routing/Router.php#L282) __toString() is used to `require` the cache file. You want to replace this with an explicit method instead of __toString() and make that method part of the interface?